src/Handler/IteratorHandler.php

```diff
@@ -124,25 +108,6 @@ public function deserializeIterator(
         return new ArrayIterator($visitor->visitArray($data, $type));
     }
 
-    /**
-     * @param mixed $data
-     */
-    public function deserializeIterable(
```

Comment:
does it mean that we do not have anymore the deserializaer for iterable?

Reply:
we do. it's inside the graph visistor/navigator itself.

Reply:
see https://github.com/schmittjoh/serializer/blob/f908d17afd08f0aab9c083322022682b41483c5b/src/GraphNavigator/DeserializationGraphNavigator.php#L137-L138

Reply:
uh, i forgot about that!